### PR TITLE
6.24 Damages & Mastery Data

### PR DESCRIPTION
**Pull request by myo**
fully tested, reviewed, and L33T approved

### CODE REVIEW

File: Orbwalking.cs [hunk 1]
```diff
@@ -1505,35 +1508,45 @@ public virtual AttackableUnit GetTarget()
...
+                        var results = (from minion in
+                            ObjectManager.Get<Obj_AI_Minion>()
+                                .Where(
+                                    minion =>
+                                        minion.IsValidTarget() && this.InAutoAttackRange(minion)
+                                        && this.ShouldAttackMinion(minion))
+                            let predHealth =
+                                HealthPrediction.LaneClearHealthPrediction(
+                                    minion,
+                                    (int) (this.Player.AttackDelay*1000*LaneClearWaitTimeMod),
```
Comment (YeezyYeezyWhatsGood) [inline Orbwalking.cs]: unreadable

Comment (myo) [inline Orbwalking.cs]: that's honda's code :keepo:

Comment (YeezyYeezyWhatsGood) [inline Orbwalking.cs]: U removed all the spaces Kappa

Comment (h3h3) [inline Orbwalking.cs]: can you stop spamming my mail inbox with useless stuff, ty!

File: Orbwalking.cs [hunk 2]
```diff
@@ -745,10 +754,13 @@ private static void OnProcessSpell(Obj_AI_Base unit, GameObjectProcessSpellCastE
             try
             {
                 var spellName = Spell.SData.Name;
```
Comment (YeezyYeezyWhatsGood) [inline Orbwalking.cs]: null reff

Comment (h3h3) [inline Orbwalking.cs]: fully tested

**Pull request merged**